myframe.cpp

```diff
@@ -498,6 +502,7 @@ void MyFrame::SetupMenuBar(void)
     tools_menu->Append(MENU_STARCROSS_TEST, _("Star-Cross Test"), _("Run a star-cross test for mount diagnostics"));
     tools_menu->Append(MENU_GUIDING_ASSISTANT, _("&Guiding Assistant"), _("Run the Guiding Assistant"));
     tools_menu->Append(MENU_DRIFTTOOL, _("&Drift Align"), _("Run the Drift Alignment tool"));
+    tools_menu->Append(MENU_POLARDRIFTTOOL, _("&Polar Drift Align"), _("Run the Polar Drift Alignment tool"));

```

Comment:
These names are too confusing now for a new user.  Maybe the names are ok, but I think we need to improve the tool-tips to explain a bit more what the differences are. Perhaps something like this (name / tool-tip):
Drift Align / alignment of the mount's polar axis by minimizing declination drift
Polar Drift Align / drift alignment using analysis of star motion with scope pointed towards the pole
Static Polar Align / alignment of the mount's polar axis using rotation about the RA axis while pointed towards the pole